runtime/bindings/python/src/compatibility/ngraph/impl/__init__.py

```diff
@@ -50,3 +50,4 @@
 from _pyngraph import Output
 from _pyngraph import DiscreteTypeInfo
 from _pyngraph import util
+from _pyngraph import Layout
```

Comment:
I think `Layout` should be exposed on a higher level as it is going to be used by `PreProcessInfo` and probably `Port` classes. Import it inside ___init.py___ for ngraph `from ngraph.impl import Layout`. Please confirm it.

Reply:
Done.